src/realm-settings/RealmSettingsSection.tsx

```diff
@@ -354,7 +373,10 @@ export const RealmSettingsSection = () => {
               aria-label="tokens-tab"
               title={<TabTitleText>{t("realm-settings:tokens")}</TabTitleText>}
             >
-              <RealmSettingsTokensTab reset={() => resetForm(realm)} />
+              <RealmSettingsTokensTab
+                realm={realm!}
```

Comment:
If the aforementioned statement is changed there is no need to use a non-null assertion here.
```suggestion
                realm={realm}
```